### PR TITLE
Update kommander air-gapped installation docs to use new airgapped flag

### PR DESCRIPTION
**Pull request by jimmidyson**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4131.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 1]
```diff
@@ -202,11 +202,15 @@ Based on the network latency between the environment of script execution and the
 
 ## Install on Konvoy
 
-1. Adapt the [configuration file][kommander-config] created from running `kommander install --init > install.yaml` for the air-gapped deployment by changing the `.apps.kommander` section. Ensure you use the actual version number everywhere `${VERSION}` appears:
+1. Create the [configuration file][kommander-config] by running `kommander install --init --airgapped > install.yaml` for the air-gapped deployment. Open the `install.yaml` file and validate that it looks similar to below, with `${VERSION}` replaced with the actual version number:
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/install/air-gapped/index.md:205]:
```suggestion
1. Create the [configuration file][kommander-config] by running `kommander install --init --airgapped > install.yaml` for the air-gapped deployment. Open the `install.yaml` file and review that it looks like the following, with `${VERSION}` replaced with the actual version number:
```

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.2/install/air-gapped/index.md (hunk 1 above)
Comment (jimmidyson) [reply pages/dkp/kommander/2.2/install/air-gapped/index.md:205]: :+1:

**Comment by jimmidyson** [comment]
Do not merge until https://github.com/mesosphere/kommander-cli/pull/364 is merged.

**Comment by msdolbey** [comment]
Heeeeey -- umm, so @jimmidyson - I used this opportunity of looking at this PR as a chance to git rid of some markdown exemptions and lint this page.

Looks good, btw! Like this feature!

**Review by msdolbey — APPROVED** [review]
👍 

True Facts

**Pull request merged**